crates/loans/src/lib.rs

```diff
@@ -1484,10 +1486,10 @@ impl<T: Config> Pallet<T> {
 
         // The liquidator may not repay more than 50%(close_factor) of the borrower's borrow balance.
         let account_borrows = Self::current_borrow_balance(borrower, liquidation_asset_id)?;
-        let account_borrows_value = Self::get_asset_value(liquidation_asset_id, account_borrows)?;
-        let repay_value = Self::get_asset_value(liquidation_asset_id, repay_amount)?;
+        let account_borrows_value = Self::get_asset_value(liquidation_asset_id, account_borrows)?.amount();
+        let repay_value = Self::get_asset_value(liquidation_asset_id, repay_amount)?.amount();
 
-        if market.close_factor.mul_ceil(account_borrows_value.into_inner()) < repay_value.into_inner() {
+        if market.close_factor.mul_ceil(account_borrows_value) < repay_value {
```

Comment:
Would it make sense to use this? Pros: we don't need to unwrap with `amount()`, and we'd have the currency check in the `lt`. Cons: less efficient. @gregdhill I'd like your input on this as well

https://github.com/interlay/interbtc/blob/8a2c8739df5c40675285321cc6332209a9696ca8/crates/currency/src/amount.rs#L145

Reply:
I haven't changed this, up to you if you want me to include

Reply:
Would probably be cleaner, I could go either way tbh